demo/demo_vgg16.ipynb

```diff
@@ -3,19 +3,22 @@
   {
    "cell_type": "code",
    "execution_count": 1,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "#importing the necessary dependencies\n",
-    "\n",
-    "from caffe.proto import caffe_pb2\n",
+    "import os\n",
     "from layers import Fixations\n",
-    "from urllib import urlretrieve\n",
+    "from urllib.request import urlretrieve\n",
```

Comment:
Change to `from urllib import urlretrieve`, the code was for python2.7 and the current syntax is for python > 3